Dockerfile

```diff
@@ -9,16 +9,15 @@ RUN yum update -y && \
     bzip2 \
     epel-release \
     gcc-c++ \
-    openssl-devel
+    openssl-devel \
+    wget \
+    unzip
 
-RUN git clone https://github.com/rbenv/rbenv.git ~/.rbenv
-RUN echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
-RUN echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+RUN wget https://cache.ruby-lang.org/pub/ruby/2.7/ruby-${ruby_version}.zip
+RUN unzip ruby-${ruby_version}.zip
+RUN ./ruby-${ruby_version}/configure
+RUN make
+RUN make install
```

Comment:
This leaves the root directory of the container very messy. It would be beneficial to download and unpack the archive in to a dedicated directory such as `/tmp/ruby-${ruby_version}`